src/coord/src/catalog/builtin.rs

```diff
@@ -645,12 +645,22 @@ pub const MZ_PERF_DEPENDENCY_FRONTIERS: BuiltinView = BuiltinView {
     sql: "CREATE VIEW mz_perf_dependency_frontiers AS SELECT DISTINCT
```

Comment:
Can you check with Andi and/or Frank on this? It's possible that `mz_perf_dependency_frontiers` was supposed to represent something specific that we are breaking now.

If that's the case, it would be better to have the "how far behind is a dataflow from all of its sources" be a separate query.

Reply:
I'm ok with modifying the definition of `mz_perf_dependency_frontiers` in this way. Arguably this is restoring what `mz_perf_dependency_frontiers` used to hold before views were no longer materialized by default.

Reply:
yeah this is much closer to what the semantic goal of this view was supposed to be. I think we couldn't do it this way before because we didn't have the necessary sql features?